File: benchmarks/cross/progs/regex.py
import re
toplam = 0
for _ in range(3000):
    bulunan = re.findall(r"[a-z]+\d+", "user1 kai@lovax.dev x99 mail: eda@oyun.io ve son: a1b2c3 v12")
    toplam += len(bulunan)
print(toplam)

File: benchmarks/cross/progs/regex.js
let toplam = 0;
for (let i = 0; i < 3000; i++) {
    const bulunan = "user1 kai@lovax.dev x99 mail: eda@oyun.io ve son: a1b2c3 v12".match(/[a-z]+\d+/g) || [];
    toplam += bulunan.length;
}
console.log(toplam);
